common/settings/settings-provider.ts

```diff
@@ -65,6 +65,7 @@ export const defaultSettings: AsbplayerSettings = {
         decreasePlaybackRate: { keys: isMacOs ? '⇧+⌃+[' : 'ctrl+shift+[' },
         increasePlaybackRate: { keys: isMacOs ? '⇧+⌃+]' : 'ctrl+shift+]' },
         toggleSidePanel: { keys: '`' },
+        toggleRepeat: { keys: 'R' },
```

Comment:
Can this shortcut be `Shift + R` instead so that it's harder to trigger by accident?

Reply:
Changed